introspection/nm-manager.xml

```diff
@@ -381,6 +381,19 @@
     <property name="Version" type="s" access="read"/>
 
     <!--
+        Capabilities:
+
+        The current set of capabilities.  This is a list of 32 bit
+        integers where specific bits are set when a capability is
+        currently available.  For example, capability number 42 would
+        be represented by bit (42 % 32) == 10 in the element with index
+        (42 / 32) == 1 in the array.
+
+        See <link linkend="NMCapability">NMCapability</link> for currently defined capability numbers.
+    -->
+    <property name="Capabilities" type="au" access="read"/>
```

Comment:
> On the other hand, I was wondering if we want to reserve a range of capabilities for downstream users. In that case, those capabilities would have a high number, and could not be easily expressed in a bitfield. That is the reason why I was thinking about exposing every capability as individual number. Hm, probably the bitfield is nicer to use and more efficient. What are other opinions here?

I don't think that efficiency should matter much (actually I first thought to represent each capability as string and return an array of strings). I like the idea of using an array of u32 where each element is a capability which can assume any value. In this way we can also easily group them for class (for example device capabilities, settings?, etc.).